src/main/java/co/rsk/bitcoinj/script/RedeemScriptValidator.java

```diff
@@ -34,32 +35,40 @@ protected static boolean hasStandardRedeemScriptStructure(List<ScriptChunk> chun
                 return false;
             }
 
-            // First chunk must be an OP_N
-            if (!isOpN(chunks.get(0))) {
+            // First chunk must be a number for the threshold
+            ScriptChunk firstChunk = chunks.get(0);
+            if (!isN(firstChunk)) {
```

Comment:
If I understand correctly, this previously only accepted values up to 15 and now it can go higher. We should add tests for this new possibilities

Reply:
https://github.com/rsksmart/bitcoinj-thin/pull/175